README.md

```diff
@@ -56,59 +53,44 @@ See [Getting Started with Google Maps Platform](https://developers.google.com/ma
  _Note: Google Maps Platfom requires creation of a billing account, but [they offer](https://cloud.google.com/maps-platform/pricing/) $200 of free monthly usage, which is enough to support development._
 
 
-### Development & Fill the Database
-
-You will need to execute migration process, run
-```
-scripts/update
-```
-then reset database and repopulate with fixture data that including parse & geocode processing, run
-```
-scripts/reset_database
+### Kick-off & start local development
+- Install node and create database structure
 ```
-Re-run & build Docker containers
+./scripts/update
 ```
-docker compose -f docker-compose.dd.yml up --build -d
+- Polulate database with seeded facility lists
 ```
-Run Docker containers on Dedupe Hub before “match” process
+./scripts/reset_database
 ```
-docker compose up -d
+- Start deduplication of seeded lists to create new facilities
 ```
-then get inside the Django container by executing
+./scripts/manage matchfixtures
 ```
-docker exec -it opensupplyhub-django-1 /bin/bash
+- Start Docker containers
 ```
-and execute a command to match or create facilities
+docker compose up
 ```
-./manage.py matchfixtures
+- Now you are ready for quick start the app
 ```
-then run two commands to index & fill facilities data
+open http://localhost:6543
 ```
-./manage.py index_facilities_new
-./manage.py fill_raw_json
-```
-Now you are ready for quick start the app.
 
 ### Restore the DB dump in the local Docker DB container
 
 1. The project containers must be running locally.
-2. Increase Django timeout to have enough time for rebuilding Gazeteer. (docker-compose.yml -> django -> command -> - "--timeout=90" or /django/-> Dockerfile -> CMD -> "--timeout=60", \)
-3. Download prod dump file
+2. Download prod dump file
+3. Place it in `[path-to-project]/dumps/` folder
 4. Then run in the terminal of your machine
 ```
-docker exec -i opensupplyhub-database-1 pg_restore --verbose --clean --no-acl --no-owner -d openapparelregistry -U openapparelregistry < /path/on/your/machine/latest_prod.dump
+docker compose run database --rm --entrypoint pg_restore -d openapparelregistry ./dumps/db.dump -c -U openapparelregistry
```

Comment:
I verified. This command doesn't work. I would leave the previous command but apply [changes](https://spdfn.slack.com/archives/C04RGFZC1GW/p1707470972791779?thread_ts=1707462211.802249&cid=C04RGFZC1GW) that were proposed by Vadim.

Reply:
@vladsha-dev If this doesn't work it needs to be fixed. 

Also could you please mention changes here. Please note that this is public repository and not everyone has access to the slack

Reply:
Just change the name of the container to the correct one in the initial command.

Reply:
> Just change the name of the container to the actual one in the initial command.

@vladsha-dev how exactly. Name of container is can be different?

Reply:
Apologies for any potential confusion. What I meant to do was:
docker exec -i **open-supply-hub-database-1** pg_restore --verbose --clean --no-acl --no-owner -d openapparelregistry -U openapparelregistry < /path/on/your/machine/latest_prod.dump